packages/core/src/context.ts

```diff
@@ -20,6 +20,7 @@ export type Context = {
   luckysheetfile: Sheet[];
   defaultcolumnNum: number;
   defaultrowNum: number;
+  adddefaultrows: number;
```

Comment:
It's better to use lowerCamelCase when adding new arguments

Reply:
Am planning to rename the variable to addDefaultRows. Please suggest me

Reply:
That's right